bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -725,6 +725,21 @@ impl Client {
         Ok(Arc::new(Room::new(room)))
     }
```

Comment:
I'm not sure if there's any benefit of leaving this `join_room_by_id` method here or not. It seems redundant to me having both, but then the SDK supports both so 🤷‍♂️

Reply:
Well the spec supports both so we have it in the SDK.

Reply:
Sorry yes, I'm just not sure if there's an advantage of using this one through the FFI and was thinking "out loud". Will leave them both - the docs will help clarify.

Reply:
Yeah, I mean somebody else might be using the bindings and might find it useful. So I think it makes sense to just leave it there.